mount/default.conf

```diff
@@ -21,16 +21,34 @@
 # Use cvmfs_config showconfig to get the effective parameters.
 #
 
+# The client profile selects configuration options tuned for certain use cases
+# Overwrite this parameter in /etc/cvmfs/default.local
```

Comment:
Isn't this comment redundant, since the first line of the file already says to do that for everything in this file?